src/index.js

```diff
@@ -41,26 +34,29 @@ module.exports = function fileLoader(content) {
   let outputPath = '';
   if (config.outputPath) {
     // support functions as outputPath to generate them dynamically
-    outputPath = (
-      typeof config.outputPath === 'function' ? config.outputPath(url) : config.outputPath
-    );
+    outputPath = typeof config.outputPath === 'function' ? config.outputPath(url) : config.outputPath;
   }
 
   const filePath = this.resourcePath;
   if (config.useRelativePath) {
-    const issuerContext = this._module && this._module.issuer
-      && this._module.issuer.context || context; // eslint-disable-line no-mixed-operators
-    const relativeUrl = issuerContext && path.relative(issuerContext, filePath).split(path.sep).join('/');
+    const issuerContext = (this._module && this._module.issuer && this._module.issuer.context) || context; // eslint-disable-line no-mixed-operators
```

Comment:
Maybe put it as `(this._module && this._module.issuer && this._module.issuer.context) ? this._module.issuer.context : context`? It seems a little confusing if it gives a context or a boolean in the fist positive case.

Reply:
I just copied this from the upstream module. I can change it, but would rather try to keep in-sync for the time being: https://github.com/webpack-contrib/file-loader/blob/master/src/index.js#L16

Reply:
I see, it's a little thing, we can keep it since it's what it is in the upstream